meta-luneos/conf/distro/include/luneos-preferred-versions.inc

```diff
@@ -24,3 +24,9 @@ PREFERRED_VERSION_gst-devtools              = "${GSTREAMER_VERSION}"
 PREFERRED_VERSION_gst-validate1.0           = "${GSTREAMER_VERSION}"
 # gst-examples is not kept in sync with the rest of gst-* recipes
 # PREFERRED_VERSION_gst-examples              = "${GSTREAMER_VERSION}"
+
+# The dt-schema utilities to check kernel's DTB does not yet
+# support python3-jsonschema 4.18+, where the behavior has changed sensibly.
+# https://github.com/devicetree-org/dt-schema/issues/108
+PREFERRED_VERSION_python3-jsonschema        = "4.17.3"
```

Comment:
This isn't needed, the version from meta-luneos will be used because of higher layer priority.

Move the comment to the recipe, so it's clear when it can be dropped to use newer version from oe-core again.